eng/Tools/DiagConfig/DiagConfig.targets

```diff
@@ -0,0 +1,89 @@
+﻿<Project>
```

Comment:
Feels as though it might be simpler and cleaner to have a separate .csproj file for running this logic, distinct from the project file to build DiagConfig itself.

We load all the analyzers anytime we build a project, so there wouldn't need to be any special logic for restore things, msbuild would just take care of that, I think.